shell/dialog/DeveloperLoadExtensionDialog.vue

```diff
@@ -150,63 +163,62 @@ export default {
 </script>
 
 <template>
-  <app-modal
+  <!-- <app-modal
```

Comment:
Let's delete the unused `app-modal` component instead of commenting it out.

Reply:
Sorry mate. Sloppy. Done